### DISCUSSION TITLE
SriLanka users cannot register with Vercel

### DISCUSSION THREAD

**Discussion by tweerasinghe** [vercel/community#4358] [Help] [answered]
Hello,
I live in Sri Lanka. To register with Vercel, I need to verify my phone number. But SriLanka does not support telephone numbers. How can I solve this issue?

**Comment by swarnava** [comment, marked as answer]
Please email us at registration@vercel.com

**Reply by tweerasinghe** [reply]
@swarnava almost 2 weeks. no replay
